src/client/datascience/jupyter/kernels/kernel.ts

```diff
@@ -68,7 +65,6 @@ export class Kernel implements IKernel {
     private readonly _onStatusChanged = new EventEmitter<ServerStatus>();
     private readonly _onRestarted = new EventEmitter<void>();
     private readonly _onDisposed = new EventEmitter<void>();
-    private readonly _onInterruptTimedOut = new EventEmitter<void>();
```

Comment:
This wasn't required, as the place this was required is the same code that starts the interrupt & it has information whether interrupt timed out or not. 
Hence we can just use the return value. Else we have two ways to determine if a kernel interrupt timeodout - check return value or wait for event.